
uint16 rc_throttle
uint16 rc_roll
uint16 rc_pitch
uint16 rc_yaw

